tools/wptrunner/wptrunner/browsers/firefox.py

```diff
@@ -102,6 +101,9 @@ def executor_kwargs(test_type, server_config, cache_manager, run_info_data,
             "network.dns.localDomains": ",".join(hostnames)
         }
         capabilities = {"moz:firefoxOptions": fxOptions}
```

Comment:
For consistency, this should be `capabilities["moz:firefoxOptions"] = fxOptions`